server/etcdserver/api/membership/cluster_test.go

```diff
@@ -1048,12 +1048,6 @@ func TestAddMemberSyncsBackendAndStoreV2(t *testing.T) {
 			name:           "Adding member should succeed if it was only in backend",
 			backendMembers: []*Member{alice},
 		},
-		{
```

Comment:
Reason for this test was to confirm that we will still return error when we try to add an existing member to cluster. The bug came from fact that we forgot about learners and setting attributes, however I think we should have an equivalent test.

Reply:
updated